### PR TITLE
doc: added doc.txt to storage test

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

he doc.txt was added to the serialization test module where it is explained how to execute it

### CODE REVIEW

File: tests/storage/doc.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+This component was tested on a samr21 board that provides us with 1024 pages where data can be saved.
+
+To test the code from python:
+
+```c
+make flash test
+
+```
+
+to test only C code"
```
Comment (RocioRojas) [inline tests/storage/doc.txt:20]: to test only C code:

File: tests/storage/doc.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+/**
+@defgroup storage_test Storage test mdoule
+
```
Comment (luisan00) [inline tests/storage/doc.txt:4]: define the oxygen group where this test is

File: tests/storage/doc.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Test Storage
+
+Here all the tests of the storage module are executed.
+
+storage uses flash memory to store data, currently only string type data can be stored and uin8_t the maximum amount that can be stored in this component is 16 bytes.
+
+This component was tested on a samr21 board that provides us with 1024 pages where data can be saved.
+
+To test the code from python:
```
Comment (luisan00) [inline tests/storage/doc.txt:13]: Modes are interactive or automatic test

File: tests/storage/doc.txt (hunk 1 above)
Comment (luisan00) [reply tests/storage/doc.txt:20]: anyway the `c` code is tested both ways

**Review by luisan00 — CHANGES_REQUESTED** [review]
reviewed!

**Review by eduazocar — APPROVED** [review]
Good job :+1:

**Review by luisan00 — APPROVED** [review]

**Pull request merged**